src/installer.ts

```diff
@@ -180,6 +224,27 @@ async function determineVersion(version: string): Promise<string> {
   return await getLatestVersion(version);
 }
 
+async function getLatestCommit(branch: string): Promise<string> {
+  let url: string = util.format('https://go.googlesource.com/go/+/refs/heads/%s?format=JSON', branch);
+  //
+  // We don’t use restm.RestClient because Gitiles returns )]}'
+  // garbage on the first line of (invalid) JSON response.
```

Comment:
The bad JSON is on purpose; I would recommend linking to the documentation for this: https://gerrit-review.googlesource.com/Documentation/rest-api.html#output